server/config.js

```diff
@@ -1,6 +1,17 @@
 import { join } from 'path'
 import { existsSync } from 'fs'
 
+require('babel-register')({
```

Comment:
👍  This is awesome.

But I feel it's kind of complicated that `next.config.js` is transpiled using the default babel settings even if you customized it.
So it might be better not to transpile this file. Node.js supports almost all useful ES6 features now anyway.